.devcontainer/devcontainer.json

```diff
@@ -26,16 +29,39 @@
 				"GitHub.copilot",

```

Comment:
Since this requires subscription, best to not have it included by default

Reply:
okay, I will remove it then.